test-util/config.ts

```diff
@@ -89,3 +86,47 @@ function isPortAvailable(port: number): boolean {
     throw error;
   }
 }
+
+function getRandomPort(min = 49152, max = 65534): number {
+  let randomPort;
+
+  do {
+    randomPort = Math.floor(Math.random() * (max - min + 1) + min);
+  } while (!isPortAvailable(randomPort));
+
+  return randomPort;
+}
+
+interface WaitForPortOptions {
```

Comment:
These options don't seem to be supplied to the `waitForPort` call. Any reason to keep this declaration and the optional param on 107?

Reply:
You are right.
These options can be re-added when needed, so I'll remove them.